R/Seurat_III.R

```diff
@@ -512,17 +525,23 @@ RemoveCellCycle <- function(seuratObj, runPCAonVariableGenes = F, pcaResultFile
     seuratObj <- RunPCA(object = seuratObj, pc.genes = VariableFeatures(object = seuratObj), do.print = F, verbose = F)
   }
 
-  #Eisa: is this really correct to repeat regression here?
-  if (!useSCTransform) {
-    seuratObj <- ScaleData(object = seuratObj, features = rownames(x = seuratObj), vars.to.regress = c("nCount_RNA", "p.mito"), display.progress = F, verbose = F)
-  } else {
-    seuratObj <- SCTransform(seuratObj, vars.to.regress = c("nCount_RNA", "p.mito"), verbose = FALSE)
-  }
+  # #Eisa: is this really correct to repeat regression here?
```

Comment:
Can you please remove the commented out code, and also our comment conversation?  I put the initial comment in this PR, but I would generally try to remove all things like this before merging into Dev.

Reply:
Also, if features is null, doesnt it already default to everything?  per docs; "Vector of features names to scale/center. Default is all features".  Your addition doesnt hurt anything, but it's the same result, right?

Reply:
On your first comment, No problem! 
On the second, Scaling used to be defaulting to variable genes for speed, this might be new and/or specific to SCT.